packages/moonstone/VideoPlayer/MediaControls.js

```diff
@@ -625,6 +633,7 @@ const MediaControlsDecorator = hoc((config, Wrapped) => {
 			this.paused = new Pause('VideoPlayer');
 
 			this.state = {
+				isFirstReadInfoComponents: true,
```

Comment:
How about `isFirstToReadInfoComponents`?

Reply:
I changed the variable name based on the discussion result.
isFirstToReadInfoComponents -> hasReadInfoComponents.